lib/pure/mimetypes.nim

```diff
@@ -8,12 +8,12 @@
 #
 
 ## This module implements a mimetypes database
-import strtabs
+import tables
 from strutils import startsWith, toLowerAscii, strip
 
 type
   MimeDB* = object
-    mimes: StringTableRef
+    mimes: Table[string, string]
 
 const mimes* = {
```

Comment:
Shouldn't this be private field? Except polluting the documentation with many entries, it shouldn't be used directly

Reply:
Why not? It's more likely we got the ``const`` right than the API...